charts/internal/gcp-infra/templates/main.tf

```diff
@@ -55,7 +55,11 @@ resource "google_compute_router_nat" "nat" {
   router =  "${google_compute_router.router.name}"
   {{ end -}}
   region                             = "{{ required "google.region is required" .Values.google.region }}"
-  nat_ip_allocate_option             = "AUTO_ONLY"
+  nat_ip_allocate_option             = {{ if .Values.networks.cloudNAT.natIPNames }}"MANUAL_ONLY"{{ else }}"AUTO_ONLY"{{ end }}
```

Comment:
Can you specify the new Helm chart values in the `values.yaml` to show what is possible to be configured?

Reply:
Yes, we can do that.